docs/resources/client/index.md

```diff
@@ -0,0 +1,15 @@
+Resource Packs
+==============
+
+[Resource Packs][respack] allow for the customization of client resources through the `assets` directory. This includes textures, models, sounds, localizations, etc. Forge, and your mod, can also have resource packs. Any user can therefore modify all the textures, models, and other asetts defined within this directory.
+
+### Creating a Resource Pack
+Resource Packs are stored within the `assets` directory within your project's resources.
+Your mod can have multiple asset domains, since you can add or modify already existing resource packs, like vanilla's, forge's, or another mod's.
+You can then follow the steps found [here][createrespack] to create any resource pack.
```

Comment:
Perhaps it may be better (partly for accessibility) to change the link text to describe where it links rather than a generic `here`:
```suggestion
You can then follow the steps found [at the Minecraft Wiki][createrespack] to create any resource pack.
```